resource-definitions/terraform-driver/credentials/aws-provider-credentials.yaml

```diff
@@ -3,20 +3,20 @@ kind: Definition
 metadata:
   id: aws-provider-credentials
 entity:
+  # Use the same account as the config we're ma
+  driver_account: ${resources['config.default#aws-account'].account}
   driver_inputs:
-    secret_refs:
-      variables:
-        # Using the "value" element of secret references to write the values.
-        aws_access_key_id:
-          value: ${resources.config#aws-account.outputs.aws_access_key_id}
-        aws_secret_access_key:
-          value: ${resources.config#aws-account.outputs.aws_secret_access_key}
     values:
+      credentials_config:
+        variables:
+          access_key_id: "AccessKeyId"
+          secret_access_key: "SecretAccessKey"
+          session_token: "SessionToken"
```

Comment:
This variable `session_token` is not used in the scripts.

Reply:
Good spot. Fixed.